Tenant-Management-Node/routes/houses.js

```diff
@@ -0,0 +1,112 @@
+var express = require('express');
+var router = express.Router();
+
+// Get Houses
```

Comment:
Kindly create a base object in this format.
apiResponse{
success: boolean;
stackTrace:string;
response: object;
}

-> if success property is true, api communication is success we can use the response to show in the UI. If success property is false due to 400 or 404 response. we can show the UI accordingly. So to handle exceptional error scenario create an object like in the above mentioned format & send that as response.

Note : 
success property - represents whether api communication is success / failure
stackTrace property : "If you need to send any message to the front end, you can use this property"
response property : this property holds the response for the particular api call.